### PR TITLE
Ocaml5.0

### PR DESCRIPTION
**Pull request by fraidev**

### CODE REVIEW

File: lib/client.ml [hunk 1]
```diff
@@ -1,29 +1,31 @@
-open Lwt.Syntax
 open Stdint
 
 type t =
-  { in_ch : Lwt_io.input_channel
-  ; out_ch : Lwt_io.output_channel
+  { flow :
```
Comment (anmonteiro) [inline lib/client.ml:4]: This can probably just be `Eio.Flow.two_way` or `Eio_unix.socket`

File: lib/tcp.ml [hunk 2]
```diff
@@ -1,11 +1,25 @@
...
+    let ip = Ipaddr.V4.to_octets host in
+    let addr = `Tcp (Eio.Net.Ipaddr.of_raw ip, port) in
+    Eio.Net.connect ~sw (Eio.Stdenv.net env) addr
   ;;
 
-  let write_bytes oc buf =
-    let buf_len = Bytes.length buf in
-    Lwt_io.write_from_exactly oc buf 0 buf_len
+  let write_bytes socket_flow buf =
+    Eio.Buf_write.with_flow socket_flow (fun buf_write ->
```
Comment (talex5) [inline lib/tcp.ml:9]: Note: this creates a new buffered writer every time you write, which is probably not what you want!

If you don't need buffering, `Flow.copy_string` would be more efficient.

File: lib/client.ml (hunk 1 above)
Comment (fraidev) [reply lib/client.ml:4]: Thank you! Fixed with this casting https://github.com/fraidev/tornado/commit/b2ce02fa69291c59c489d0de98c22c01a4153871

File: lib/tcp.ml (hunk 2 above)
Comment (fraidev) [reply lib/tcp.ml:9]: Thank you! I did this fast just to pass the tests and forgot to edit it haha.
Fixed with https://github.com/fraidev/tornado/commit/9c3a6ab2e10441acb96cd586cd9b4ec847bbe94f

**Pull request merged**